packages/core/src/Stores/client-store.js

```diff
@@ -335,9 +335,25 @@ export default class ClientStore extends BaseStore {
     }
 
     @computed
-    get has_account_error_in_mt5_list() {
-        if (!this.is_logged_in) return false;
-        return this.mt5_login_list?.some(account => !!account.has_error);
+    get list_of_real_mt5_accounts_with_error() {
+        if (!this.is_logged_in) return [];
+        return this.mt5_login_list?.filter(account => !!account.has_error && account.account_type === 'real');
+    }
```

Comment:
(suggestion) Can we refactor the logic of this computed property, and the logic inside `list_of_demo_mt5_accounts_with_error` to a pure function here?